tests/test-helpers/src/builder.rs

```diff
@@ -210,7 +211,7 @@ impl TestSetupBuilder {
 
         let latest_block = self.starting_block.map(|starting_block| LastBlockConfig {
             block_height: starting_block,
-            state_transition_version: 0,
+            state_transition_version: LATEST_STATE_TRANSITION_VERSION - 1,
```

Comment:
Is this related to the other changes? Why are we pointing to the "old" version here?

Reply:
Yeah, it is not related to the change, just saw in the codebase=D We want to use previous version because after regenesis we want to use a native executor just to speed up tests.